environments/network/demo.tfvars

```diff
@@ -64,7 +64,22 @@ additional_routes_appgw = [
 ]
 
 additional_routes_coreinfra = [
+  {
+    name                   = "nonprodHub"
+    address_prefix         = "10.0.0.0/8"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.72.36"
+  },
+  {
+    name                   = "ptl"
+    address_prefix         = "10.10.72.0/21"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.72.36"
+  },
```

Comment:
bit different to other envs but it matches the existing manual config for the other subnets